### DISCUSSION TITLE
The depth-curve in the example

### DISCUSSION THREAD

**Discussion by hcoludfly** [SynxFlow/SynxFlow#36] [Q&A] [answered]
Hi, I have a question when I run the example: why does the depth profile stay the same after 2000s, I think it may decrease to 0

**Comment by xiaxilin** [comment, marked as answer]
This is because the model can extrapolate the discharge input. In the example, the discharge remains constant during the simulation.

**Reply by hcoludfly** [reply]
Thank you for your reply, I have another question: how should I modify it to get a plot of discharge over time instead of depth?

**Reply by xiaxilin** [reply]
Hi, this can be done by changing the file tag to hU - which is the unit-width discharge at the gauging point

```python
gauges_pos, times, values = case_output.read_gauges_file(file_tag = 'hU')
```

**Reply by hcoludfly** [reply]
Thank you very much, your answer is extremely useful to me.
